test/integration/api/forms/draft.js

```diff
@@ -1167,7 +1169,9 @@ describe('api: /projects/:id/forms (drafts)', () => {
                           formDefId: form.draftDefId,
                           name: attachment.name,
                           oldBlobId: attachment.blobId,
-                          newBlobId: attachment2.blobId
+                          newBlobId: attachment2.blobId,
+                          oldDatasetId: attachment.datasetId,
```

Comment:
This is probably fine, just thinking about how it will show up in the audit logs and if people will wonder about dataset IDs showing up for all their attachment changes.

Or maybe i'm thinking about the fact that these tests aren't about dataset attachments so the dataset IDs will always be null. We might want more tests of what it looks like when the attachments are updated to link to datasets.